public/css/app.css

```diff
@@ -0,0 +1,160 @@
+*{
+    --dark:#0d0d0d;
+    --dark-gray:#404040;
+    --tertiary:#BABF24;
+    --secondary: #13F2A1;
+    --primary: #6DF2CC;
+    --gray: #c0c0c0;
+    --white: #f2f2f2;
+    --danger: #F26D6D;
+}
+
+@font-face {  
+    font-family: "Sofia Sans", sans-serif;
+    src: url('/fonts/SofiaSans-Regular.ttf');
+    font-weight: normal;  
+    font-style: normal;
+}
+    
+
+
+body{
+    font-family: "Sofia Sans", sans-serif;
+    background-color: var(--dark);
+    color: var(--white);
+}
+
+.text-effect{
+    border: 1px solid;
+    text-shadow: -3px 3px var(--primary), -6px 6px var(--secondary), -9px 9px var(--tertiary);
+}
+
+.input-login{
+    background-color: var(--white);
+    color: var(--dark);
+    border: 1px solid var(--primary);
+    border-radius: 10px;
+    width: 600px;
+    height: 45px;
+}
+
+.input-nav-small{
+    background-color: var(--white);
+    color: var(--dark);
+    border: 1px solid var(--gray);
+    border-radius: 20px;
+    background-image: url(https://cdn2.hubspot.net/hubfs/4004166/bioticresearch_website_assets/images/search_icon.png);
+    background-repeat: no-repeat;
+    background-position: right center;
+    width: 340px;
+    height: 25px;
+}
+
+.input-nav-default{
+    background-color: var(--white);
+    color: var(--dark);
+    border: 1px solid var(--gray);
+    border-radius: 20px;
+    background-image: url(https://cdn2.hubspot.net/hubfs/4004166/bioticresearch_website_assets/images/search_icon.png);
+    background-repeat: no-repeat;
+    background-position: right center;
+    width: 200px;
+    height: 25px;
+}
+
+.input-nav-larger{
+    background-color: var(--white);
+    color: var(--dark);
+    border: 1px solid var(--gray);
+    border-radius: 20px;
+    background-image: url(https://cdn2.hubspot.net/hubfs/4004166/bioticresearch_website_assets/images/search_icon.png);
+    background-repeat: no-repeat;
+    background-position: right center;
+    width: 668px;
+    height: 25px;
+}
+
+.button-default{
+    width: 94px;
+    height: 34px;
+    border: 1px solid var(--dark-gray);
+    background-color: var(--secondary);
+    border-radius: 5px;
+    
+}
+
+.button-default:hover{
+    width: 94px;
+    height: 34px;
+    border: 1px solid var(--dark-gray);
+    background-color: var(--primary);
+    border-radius: 5px;
+    box-shadow: -2px 2px 5px var(--white);
+}
+
+.footer{
+    background-color: var(--dark-gray);
+    color: var(--white);
+    width: 100%;
+    height: 100px;
+    
+} 
+
+.tojoja-footer{
+    font-size: 20px;
+    font-family: "Sofia Sans", sans-serif;
+    float: left;
+    padding: 15px;
+}
+
+.git-about-footer{
+    font-size: 20px;
+    font-family: "Sofia Sans", sans-serif;
+    float: right;
+    padding: 40px;
+    
+}
+
+.git-about-footer a{
+    color: var(--white);
+    text-decoration: none;
+}
```

Comment:
Acho que essas classes do footer podem ficar no componente do mesmo quando for criado, pois o footer pode mudar caso o usuario esta logado ou não